### ISSUE TITLE
about codingquarry command line

### ISSUE THREAD

**Issue by xizhesun** [nextgenusfs/funannotate#795]
**Are you using the latest release?**
yes

**Describe the bug**
codingquarry should use "-d" for the unstrand stringtie assembly. 

I didn't find the option for the strand-specific information in the funannoate pipeline. 

And the following is the CodingQuaary recommended command line.

> Recommended run for annotating a genome using transcripts derived from
> stranded RNA-seq:
> CodingQuarry -f myGenome.fa -t myTranscript.gff3 -p 8
> 
> Recommended run for annotating a genome using transcripts derived from
> un-stranded RNA-seq:
> CodingQuarry -f myGenome.fa -t myTranscript.gff3 -p 8 -d

**Comment by hyphaltip** [comment]
I don't think we record the stranded-ness information that would be provided to `train` into `predict` program so this would require some other state to be passed or require the user to also provide this info -- I'd be curious if it really makes much of a different in final prediction set but @nextgenusfs not sure if you have an opinion.

@xizhesun Have you run this manually and checked to see if the resulting prediction set differs much in the CodingQuarry results that are produced in funannotate and those which are produced with the `-d` option.

**Comment by xizhesun** [comment]
I have not manually run CodingQuarry with "-d" option.  I just see the recommended pipeline from the "CodingQuarry -h". I think it should have something different to set "-d".  Why not add a judgment for strand info of the BAM file?
